src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -71,6 +71,7 @@
     "d:transactions/breakdowns.span_ops.ops.browser@millisecond": PREFIX + 122,
     "d:transactions/breakdowns.span_ops.ops.resource@millisecond": PREFIX + 123,
     "d:transactions/breakdowns.span_ops.ops.ui@millisecond": PREFIX + 124,
+    "d:transactions/measurements.inp@millisecond": PREFIX + 125,
```

Comment:
Adding this string means that data collected before this change will be effectively lost for this metric, as any org-specific string index for this metric will now be obscured by the index defined here. Afaict this currently affects two orgs.

Reply:
Thanks for taking a look again Joris 👍 

I guess I'm wondering a bit about the purpose these predefined metrics serve if moving forward we'll likely have to have mixed pre-defined (these static ones) and ones dynamically created already. Of course changing the test works too, but is this list now frozen in time? I imagine as more and more people use measurements, most measurement names we'll come with will already have been sent in, thereby making this list no-longer updatable.

Reply:
I'll do your recommendation of changing the test code, for now though, that is just a question moving forward for this list in the metrics indexer.

Reply:
> is this list now frozen in time? I imagine as more and more people use measurements, most measurement names we'll come with will already have been sent in

Yes I think that is an accurate assessment, looking at

https://github.com/getsentry/sentry/blob/9f4ca288acbb722a39650f5c04efdaa81ae46a58/src/sentry/sentry_metrics/indexer/strings.py#L18-L19

If at some point we want to reduce storage cost by consolidating duplicate strings, I guess we'll need a migration strategy to migrate the data behind a specific string from per-org to shared. cc @onewland